### PR TITLE
Update tf.nn.softmax_cross_entropy_with_logits.md

### PR DESCRIPTION
**Pull request by XuesongYang**
wrong description of the argument `logits`

### CODE REVIEW

**Review by XuesongYang — COMMENTED** [review]
no problem
